### DISCUSSION TITLE
RHF DevTool import suggestion never shown up on VSCode

### DISCUSSION THREAD

**Discussion by thanhpd** [react-hook-form/react-hook-form#10824] [General]
Hello, I believe this question is posted at the correct place.

I'm using `@hookform/devtools` package and whenever I try to fetch the import suggestion for the `DevTool` component, it just won't show up at all. I've tried with new Vite ReactTS projects locally and the result is still the same:

![image](https://github.com/react-hook-form/react-hook-form/assets/6737911/507b5acd-873c-4c29-8c2b-e95b35b6efab)

Strangely enough, if I try with CodeSandbox (https://codesandbox.io/p/sandbox/magical-jackson-v594p3), then the entry does show up:

![image](https://github.com/react-hook-form/react-hook-form/assets/6737911/a19a4c58-8ce5-491f-9284-f54d0ffee121)

Is there a way I can force the import suggestion to appear on VSCode?

Thank you in advance.

**Comment by Moshyfawn** [comment]
Try reloading your IDE window

![rhf-devtools-vscode](https://github.com/react-hook-form/react-hook-form/assets/16290753/e61efede-9cf2-4838-b777-5a49eb9edd0d)

**Reply by thanhpd** [reply]
Hi, I've tried your suggestion, but it won't resolve the issue. I even tried to restart my machine.
I failed to see how this would address the problem since other packages can still be looked up without reloading VSCode.

**Reply by Moshyfawn** [reply]
Reloading the IDE window solves it for me. I'm using VSCode `1.82.0-insider` and I'm pretty sure the auto-imports worked in VSCode at one point.

I honestly feel like it's something to do with the latest VSCode updates and not RHF Devtools package itself.

https://github.com/react-hook-form/react-hook-form/assets/16290753/29cae534-0729-4eb7-a9b7-69c93ecb22e9

**Reply by Moshyfawn** [reply]
I'm using the new [Zed IDE](https://zed.dev/) a lot of the time and auto-import works right after I install `@hookform/devtools`
